### ISSUE TITLE
Fix layout shift when going from 0 to 1 selected checkbox

### ISSUE THREAD

**Issue by corneliusroemer** [loculus-project/loculus#3024]
What about the initial layout shift between "all" and "1 selected entry"? That might also be nice to get rid of.

_Originally posted by @corneliusroemer in https://github.com/loculus-project/loculus/pull/3020#pullrequestreview-2378139413_
            
![2024-10-18 15 51 42](https://github.com/user-attachments/assets/297dddb2-af20-46a9-bf4b-644a3a91c2d6)

**Comment by fhennig** [comment]
It's unclear to me how we should get rid of the layout shift, what exactly are you envisioning?

**Comment by chaoran-chen** [comment]
I agree with @fhennig and wouldn't call it a bug. @corneliusroemer, do you have a suggestion on how to improve?
